.vscode/settings.json

```diff
@@ -3,6 +3,7 @@
     "files.exclude": {
         "**/.git": true,
         "bazel-*": true
-    }
+    },
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
hum, I thought this was the default behavior in VSCode now

Reply:
It wasn't for me. Maybe I got grandfathered or something?

Reply:
:man_shrugging: